tests/mock.data.js

```diff
@@ -31,3 +31,12 @@ export const createRequestBody = ({issuer, vc = validVc}) => {
 export function createISOTimeStamp(timeMs = Date.now()) {
   return new Date(timeMs).toISOString().replace(/\.\d+Z$/, 'Z');
 }
+
+export const invalidCredentialSubjectTypes = new Map([
+  ['null', null],
+  ['a boolean', true],
+  ['a number', 4],
+  ['an empty Array', []],
+  ['a string', 'a string']
```

Comment:
This is going to confuse people given that these are not normative statements in the VC Data Model specification. We need to figure out a way to do negative tests w/o using normative language. If we use normative language, people are going to go searching in the specs to see where it says "MUST NOT" be one of these values, and if they don't find it, they'll raise issues on the spec or the test suite and cause more work for all of us.

We can do two things here: Use the spec "MUST be an object" language and make that the normative test, and if they fail the test, tell them why they failed the test "expecting an object and a string was found instead" or something to that effect.

Reply:
Addressed here: https://github.com/w3c-ccg/vc-api-issuer-test-suite/pull/34/commits/a3b11101a20f15c01ebdbe0e30f78739140b5124